src/globals.cpp

```diff
@@ -50,7 +50,7 @@ int BetterSMS::getScreenRenderWidth() {
     case AspectRatioSetting::WIDE:
         return 700;
     case AspectRatioSetting::ULTRAWIDE:
-        return 1050;
+        return 833;
```

Comment:
Currently, this breaks the ultra-wide screen ratio. In actuality, I believe the value returned by `WIDE` is simply wrong. Seems to be 797 is needed, and 1046 for ultra-wide.

If we are to keep this value, we need to adjust how the widescreen code re-positions elements, and also change the value returned by `WIDE`